bower.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "transportapi.js",
-  "version": "0.0.7",
+  "version": "0.1.0",
   "main": "dist/scripts/transportapi.min.js",
   "description": "A proxy API to TAPI endpoints",
-  "license": "TODO",
+  "license": "",
```

Comment:
`unlicense` then? 

Candidate text for line 6 (using https://github.com/mikegogulski/bitcoin-php as a template)
`transportapi.js is free and unencumbered public domain software. For more information, see http://unlicense.org/`